.storybook/main.js

```diff
@@ -14,4 +17,10 @@ module.exports = {
   features: {
     emotionAlias: false,
   },
+  async webpackFinal(config) {
+    config.resolve ??= {}
```

Comment:
Not supported in vanilla JS. Which breaks the build.

Reply:
:+1: Will degrade to support older nodejs versions